docs/src/pages/components/no-ssr/FrameDeferring.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import NoSsr from '@material-ui/core/NoSsr';
+
+const useStyles = makeStyles({
+  container: {
+    maxWidth: 300,
+    wordBreak: 'break-all',
+  },
+});
+
+/**
+ * Changing the return value of this component to `any`
+ * As currently typescript does not support returning array from components
+ *
+ * https://github.com/DefinitelyTyped/DefinitelyTyped/issues/20356
+ */
+function LargeTree(): any {
+  return Array.from(new Array(3000)).map((_, index) => <span key={index}>.</span>);
```

Comment:
```suggestion
  return Array.from(new Array(3000)).map((_, index) => <span key={index}>.</span>) as JSX.Element;
```

It's trickier than I thought because any comments here will appear in the JS version. I think this undocumented is a good enough compromise.

Reply:
I already tried this. It was throwing this error. 
<img width="1405" alt="Screenshot 2019-10-17 at 22 41 29" src="https://user-images.githubusercontent.com/16988745/67031506-539bff00-f12f-11e9-988d-9903c69404ee.png">

So only i changed to `any`.

Reply:
Yeah that works as well. Still need to remove the comment though.

Reply:
ok